include/seqan/stream/formatted_file.h

```diff
@@ -400,6 +400,11 @@ struct FormattedFileContext
     typedef Nothing Type;
 };
 
+template <typename TFormattedFile, typename TStorageSpec>
+struct FormattedFileContext<TFormattedFile const, TStorageSpec> :
+    FormattedFileContext<TFormattedFile, TStorageSpec>
```

Comment:
Sorry, I should have commented the line below... I'm not referring to TStorageSpec. The FormattedFileContext of a TFormattedFile const should be a FormattedFileContext::Type const.